prov/verbs/src/verbs_info.c

```diff
@@ -46,13 +46,11 @@
 
 #define VERBS_DOMAIN_CAPS (FI_LOCAL_COMM | FI_REMOTE_COMM)
 
-#define VERBS_MSG_TX_CAPS (OFI_TX_MSG_CAPS | OFI_TX_RMA_CAPS | FI_ATOMICS | \
-			   FI_HMEM)
-#define VERBS_MSG_RX_CAPS (OFI_RX_MSG_CAPS | OFI_RX_RMA_CAPS | FI_ATOMICS | \
-			   FI_HMEM)
+#define VERBS_MSG_TX_CAPS (OFI_TX_MSG_CAPS | OFI_TX_RMA_CAPS | FI_ATOMICS)
+#define VERBS_MSG_RX_CAPS (OFI_RX_MSG_CAPS | OFI_RX_RMA_CAPS | FI_ATOMICS)
 #define VERBS_MSG_CAPS (VERBS_MSG_TX_CAPS | VERBS_MSG_RX_CAPS | VERBS_DOMAIN_CAPS)
-#define VERBS_DGRAM_TX_CAPS (OFI_TX_MSG_CAPS | FI_HMEM)
-#define VERBS_DGRAM_RX_CAPS (OFI_RX_MSG_CAPS | FI_HMEM)
+#define VERBS_DGRAM_TX_CAPS OFI_TX_MSG_CAPS
+#define VERBS_DGRAM_RX_CAPS OFI_RX_MSG_CAPS
```

Comment:
These changes are undoing changes added by a previous patch.  Just update the prior patch.

Reply:
Will fix.